examples/Example_domain/datapackage.json

```diff
@@ -1,72 +1,85 @@
 {
-  "name": "Example_domain",
-  "title": "Example reef cluster data package",
-  "description": "Example data package for ADRIA",
-  "version": "0.4.0-rc",
-  "sources": [],
-  "simulation_metadata": {
-    "timeframe": [2025, 2099]
-  },
-  "contributors": [
+"name": "Test_domain",
```

Comment:
This file is still formatted incorrectly.

Two-space indentation please.

Reply:
Seems my vs code keeps resetting the indentation, I'll see wha I need to change in the settings